WcaOnRails/config/application.rb

```diff
@@ -14,6 +14,17 @@
 ENVied.require(*ENV['ENVIED_GROUPS'] || Rails.groups)
 
 module WcaOnRails
+  AVAILABLE_LOCALES = {
```

Comment:
External file seems better to me, especially when I think of more data that will be added. No reason for cluttering this file with it =)